src/react/projects/spark-react/src/SprkButton/SprkButton.test.js

```diff
@@ -55,3 +57,13 @@ it('should apply any unknown props as attributes on the element', () => {
   const wrapper = shallow(<SprkButton aria-labelledby="label-1" />);
   expect(wrapper.find('button[aria-labelledby="label-1"]').length).toBe(1);
 });
+
+it('should render an anchor tag if an href is provided and no element is', () => {
+  const wrapper = shallow(<SprkButton href="#" />);
+  expect(wrapper.find('a[href="#"]').length).toBe(1);
+});
+
+it('should render an href is one is provided', () => {
```

Comment:
Talked with Katie about this! Currently, when a dev sets the element prop to "button" and supplies an href. Our button component with render a button with an href. Which, isn't valid HTML. Katie is going to make a fix so that we only render an href if the element is not set to button or something of the like.